tests/main/interfaces-joystick/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Ensure that the joystick interface works.
+
+details: |
+    The joystick interface allows reading and writing to joystick devices.
+
+prepare: |
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-joystick
+
+    . "$TESTSLIB/files.sh"
+    ensure_file_exists_backup_real /dev/input/js31
+    ensure_file_exists_backup_real /run/udev/data/c13:31
```

Comment:
Could you pleas provide explanation for why the major minor numbers here matter and what they represent (as a comment in the code please)

Reply:
IIRC char major 13 are input devices, then the lower range was either mice or joysticks (probably joystick since the PR is about joystick interface).

Reply:
Yup, https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/devices.txt#L408 input devices, joystick.

Reply:
I meant to add this to the code as a comment.

Reply:
Comment added